modules/_labs/core/react/index.ts

```diff
@@ -1,8 +1,8 @@
 import type from './lib/type';
 import space from './lib/space';
 import CanvasProvider from './lib/CanvasProvider';
-
 export default type;
 export {type, space, CanvasProvider};
 export * from './lib/type';
+export * from './lib/StaticStates';
```

Comment:
we can either export this or just import it as `import {StaticStates} from '@workday/canvas-kit-labs-core/react/lib/StaticStates'` I'm fine with either